### PR TITLE
css: add info on automatic prefixing since 1.8

### PR DESCRIPTION
**Pull request by arthurvr**
would fix https://github.com/jquery/api.jquery.com/issues/339

### CODE REVIEW

File: entries/css.xml [hunk 1]
```diff
@@ -144,6 +144,7 @@ $( "div" ).click(function() {
...
+	  <p>Since of jQuery 1.8, the <code>.css()</code> setter will automatically take care of prefixing the property name. For example, take <code>.css( "user-select", "none" );</code> in Chrome/Safari we'il set it as <code>-webkit-user-select</code>, Firefox will use <code>-moz-user-select</code>, and IE10 will use <code>-ms-user-select</code>.</p>
```
Comment (gnarf) [inline entries/css.xml]: `As of jQuery 1.8`, or `Since jQuery 1.8` - are there particulars of how this is explained you like to expand on @mikesherov ?

Comment (arthurvr) [inline entries/css.xml]: True. Updating my PR tommorow.

**Comment by arthurvr** [comment]
@gnarf Updated PR

File: entries/css.xml [hunk 2]
```diff
@@ -144,6 +144,7 @@ $( "div" ).click(function() {
...
+      <p>As of jQuery 1.8, the <code>.css()</code> setter will automatically take care of prefixing the property name. For example, take <code>.css( "user-select", "none" )</code> in Chrome/Safari we'il set it as <code>-webkit-user-select</code>, Firefox will use <code>-moz-user-select</code>, and IE10 will use <code>-ms-user-select</code>.</p>
```
Comment (Krinkle) [inline entries/css.xml]: Typo `we'il`

Comment (arthurvr) [inline entries/css.xml]: fixed, thanks @krinkle!

File: entries/css.xml (hunk 1 above)
Comment (arthurvr) [inline entries/css.xml]: > are there particulars of how this is explained you like to expand on @mikesherov ?

@gnarf Is this PR good for now? We can always extend it later.

File: entries/css.xml [hunk 3]
```diff
@@ -144,6 +144,7 @@ $( "div" ).click(function() {
...
+      <p>As of jQuery 1.8, the <code>.css()</code> setter will automatically take care of prefixing the property name. For example, take <code>.css( "user-select", "none" )</code> in Chrome/Safari we'll set it as <code>-webkit-user-select</code>, Firefox will use <code>-moz-user-select</code>, and IE10 will use <code>-ms-user-select</code>.</p>
```
Comment (Krinkle) [inline entries/css.xml]: It was changed to `we'll` (short for `we will`). Don't you mean `will`? That's what the other sentence parts use and also makes more sense in this context.

**Comment by gnarf** [comment]
LGTM

**Pull request closed without merging**